### ISSUE TITLE
Even if `nuxt-ts build` fails, the status code will always be zero.

### ISSUE THREAD

**Issue by iwata** [nuxt/nuxt.js#4897]
### Version

[nuxt-ts](https://github.com/nuxt.js/releases/tag/nuxt-ts)

### Reproduction link

[https://github.com/iwata/nuxt.js/tree/build-bug](https://github.com/iwata/nuxt.js/tree/build-bug)

### Steps to reproduce

```sh
$ git clone git@github.com:iwata/nuxt.js.git
$ cd ./nuxt.js
$ git checkout -t origin/build-bug
$ cd examples/typescript-tsx
$ yarn install
$ yarn build
$ echo $?
```

### What is expected ?

`echo $?` will show `1`.

### What is actually happening?

`echo $?` shows `0`.

### Additional comments?

If `nuxt build` fails in v2.4.0, it will be status code 1 as expected.

<div align="right"><sub><em>This bug report is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/nuxt.js/issues/c8570">#c8570</a>)</em></sub></div>

**Comment by manniL** [comment]
Strange. https://github.com/nuxt/nuxt.js/pull/4806 should've fixed that.

**Comment by iwata** [comment]
Here is the screencast.
[![asciicast](https://asciinema.org/a/224345.svg)](https://asciinema.org/a/224345)

**Comment by njam** [comment]
I experience `nuxt-ts generate` exiting with code 0 on errors, using nuxt-ts version 2.4.3.

**Comment by EnOane** [comment]
add to nuxt.confg  

```
build: {
    useForkTsChecker: false
  }
```

**Comment by iwata** [comment]
@EnOane 
I think that it is not fundamentally settled.

**Comment by pimlie** [comment]
That error is catched [here](https://github.com/nuxt/nuxt.js/blob/dev/packages/cli/src/setup.js#L20) and it logs the error with consola.error instead of consola.fatal which means the exit code is not set. The fix in #4806 does not apply because the exception is catched and does not traverse up to that fix.

Btw @iwate, nice bug report with the screencast. Very helpful! :+1:

**Comment by iwata** [comment]
@pimlie 
Thanks for your prompt action!
I am looking forward to the release of the fixed version to NPM.

**Comment by kevinmarrec** [comment]
@iwata I'm working on a refactor which directly fixes the issue, but will only available for 2.5, so I made a hotfix which should be ready to be shipped in next patch version (`2.4.4`).

**Comment by iwata** [comment]
I tried again by v2.4.5 and fixed this issue.
Thanks!
